xep-0424.xml

```diff
@@ -120,9 +125,7 @@
 
   <example caption="The client sends out a retraction message"><![CDATA[
 <message type='chat' to='lord@capulet.example' id='retract-message-1'>
-  <apply-to id="origin-id-1" xmlns="urn:xmpp:fasten:0">
-    <retract xmlns='urn:xmpp:message-retract:0'/>
-  </apply-to>
+  <retract id="origin-id-1" xmlns='urn:xmpp:message-retract:1'/>
```

Comment:
You are changing the example here without changing the normative description a few lines above which is still pointing at XEP-0422.

Reply:
Fixed.